lib/trackler/implementation.rb

```diff
@@ -56,6 +55,8 @@ def git_url
 
     private
 
+    attr_reader :track
```

Comment:
I need to test the downstream implications of this.
It should be ok, because track was only recently added and downstream have not had a chance to use it.